jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/AutocompleteTests.java

```diff
@@ -0,0 +1,68 @@
+package io.github.epam.angular.tests.elements.complex;
+
+import io.github.epam.*;
+import org.testng.annotations.*;
+
+import static io.github.com.StaticSite.*;
+import static io.github.com.pages.AngularPage.*;
+import static io.github.epam.site.steps.States.*;
+import static org.testng.Assert.*;
+
+public class AutocompleteTests extends TestsInit {
```

Comment:
The goal of this class is to test all autocomplete functionality with jdi asserts. If any JDI assert is missing, you should implement it.

Reply:
Done.